File: Prime_number.java

package practice2;

import java.util.Scanner;
public class Prime_number {
    public static void main(String[]args){
        System.out.print("Enter a positive integer:");
        Scanner input=new Scanner(System.in);
        int n=input.nextInt();
        int i, count=0;
        for(i=2;i<n;i++){
            if(n % i == 0){
                count++;
                break;
            }
        }
         if(count==0)
             System.out.print(n+" is a prime number.");
         else
             System.out.print(n+" is not a prime number");
            
            
        
    }
    
}

File: prime number.c
#include<stdio.h>
int main()
{
    int i,n,count=0;
    printf("Enter a positive integer:")
    scanf("%d",&n);

    for(i=2;i<n;i++)
    {
        if(n%i==0)
        {
            count++;
            break;
        }
    }
    if(count=0)
        printf("%d is prime number:",n);
    else
        printf("%d is not prime number.",n);

    return 0;
}
